pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -89,7 +89,7 @@ func AddToManager(mgr manager.Manager,
 		}
 		if err := c.Watch(
 			source.NewKindWithCache(&testimagestreamtagimportv1.TestImageStreamTagImport{}, buildClusterManager.GetCache()),
-			testImageStreamTagImportHandlerForNamedCluster(buildClusterName),
+			testImageStreamTagImportHandlerForNamedCluster(buildClusterName, ignoreClusterNames),
```

Comment:
This is incorrect, we don't even get here if the cluster is in `ignoreClusterNames`.

What needs adjustment is `testImageStreamTagImportHandler`, which handles `testImageStreamTags` on app ci by creating a request using the cluster that is in them. If that cluster is in `ignoreClusterNames` we need to do nothing there